tests/Integration/Metadata/DateArchiveTest.php

```diff
@@ -0,0 +1,193 @@
+<?php
+/**
+ * Integration Tests: Date Archive pages metadata
+ *
+ * @package Parsely\Tests
+ */
+
+declare(strict_types=1);
+
+namespace Parsely\Tests\Integration\StructuredData;
+
+use Parsely\Tests\Integration\TestCase;
+use Parsely\Metadata\Date_Builder;
+use Parsely\Parsely;
+
+/**
+ * Integration Tests for Date Archive pages metadata.
+ *
+ * @see https://www.parse.ly/help/integration/jsonld
+ */
+final class DateArchiveTest extends TestCase {
+	/**
+	 * Internal variable.
+	 *
+	 * @var $date_builder Holds the Date_Builder object.
+	 */
+	private static $date_builder;
+
+	/**
+	 * Runs once before all tests.
+	 */
+	public static function set_up_before_class(): void {
+		self::$date_builder = new Date_Builder( new Parsely() );
+
+		self::factory()->post->create( array( 'post_date' => '2022-10-31 23:59:59' ) );
+	}
+
+	/**
+	 * Setup method called before each test.
+	 */
+	public function set_up(): void {
+		parent::set_up();
+
+		$this->set_permalink_structure( '/%year%/%monthnum%/%day%/%hour%/%minute%/%second%/%postname%/' );
```

Comment:
I'd suggest explicitly setting the date and time format here, so you aren't relying on WP core never changing its defaults. It will also make the tests easier to follow.